pages/burials-memorials/eligibility.md

```diff
@@ -65,21 +68,35 @@ At the time of need, the person planning the burial can also request other honor
 - [Presidential Memorial Certificate signed by the current President of the United States](/burials-memorials/memorial-items/presidential-memorial-certificates/)
 - [Headstone, marker, niche cover, or medallion to identify a gravesite located outside a VA national cemetery](/burials-memorials/memorial-items/headstones-markers-medallions/)
 
+
 [Learn about the burial planning process](/burials-memorials/plan-a-burial/). <br />
-[Learn about memorial items](/burials-memorials/memorial-items/).
+[Learn about memorials and burial honors](/burials-memorials/honor/). <br>
+[See our National Cemetery Administration brochure](https://www.cem.va.gov/cem/docs/factsheets/NCA_Brochure.pdf).
+
 
 ## If I'm the survivor of a Veteran or Servicemember, what other benefits can I get?
 
-We give the surviving spouse, children, and parents of Veterans and Servicemembers a range of benefits. These may include tax-free monetary benefits (called Dependency and Indemnity Compensation) and help paying for the funeral service (called a burial allowance).<br>
-[See if you're eligible for Dependency and Indemnity Compensation (DIC)](/burials-memorials/dependency-indemnity-compensation/). <br>
-[Find out how to apply for a burial allowance](/burials-memorials/veterans-burial-allowance/).
+We give the surviving spouse, children, and parents of deceased Servicemembers and Veterans a range of benefits. These may include tax-free monetary benefits and help paying for the funeral service. <br>
+[Learn about survivor benefits](/burials-memorials/survivor-and-dependent-benefits/).
```

Comment:
I don't think `/burials-memorials/survivor-and-dependent-benefits.md` exists either.